src/seedu/addressbook/parser/Parser.java

```diff
@@ -99,6 +89,9 @@ public Command parseCommand(String userInput) {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
+        case SortCommand.COMMAND_WORD:
+            return new SortCommand();
+
         case HelpCommand.COMMAND_WORD: // Fallthrough
```

Comment:
rmb to go into HelpCommand and update the help command to include sort command message usage as well (: